protocol/src/test/kotlin/tbdex/sdk/protocol/ValidatorTest.kt

```diff
@@ -73,18 +73,29 @@ class ValidatorTest {
     val stringRfqWithoutPayinAmount = """
       {
         "metadata": {
-          "from": "did:key:z6MkpkvGVrxxTVbo56mvbSiF6iCKNev56wqoMcHHowqUqvKQ",
-          "to": "did:ex:pfi",
+          "from": "did:dht:9kkuh34q7nkd4tphbcg7py9h1g16iftbtskesi9courdwj96q3sy",
+          "to": "did:dht:8rqqxczxhdugndj5mykiahy7y4zg4zkk3jajr6qyo5owdrgqqx3y",
+          "protocol": "1.0",
           "kind": "rfq",
-          "id": "rfq_01hkx53kgafbmrg2xp87n5htfb",
-          "exchangeId": "rfq_01hkx53kgafbmrg2xp87n5htfb",
-          "createdAt": "2024-01-11T20:58:34.378Z",
-          "protocol": "1.0"
+          "id": "rfq_01ht6fh3fsf9z8ekh5tyjbsmwk",
+          "exchangeId": "rfq_01ht6fh3fsf9z8ekh5tyjbsmwk",
+          "createdAt": "2024-03-30T01:28:03.321Z"
         },
         "data": {
-          "offeringId": "abcd123",
+          "offeringId": "offering_01ht6fh3fre7t8zfc6j97t2pe1",
           "payin": {
             "kind": "DEBIT_CARD",
+            "paymentDetailsHash": "k8nU7MKYNb140u3pN8FO5ndAK3WDJ8lQDEsYA0H9fWA"
+          },
+          "payout": {
+            "kind": "BTC_ADDRESS",
+            "paymentDetailsHash": "xwyJqXOcLsYBBojNlVHZGZzlhZvswFPUh3xsLxus_EU"
+          },
+          "claimsHash": "o16anc251kJb3uyYKzPdlS5SdhGomylHZ8_YZ7CBbgU"
+        },
+        "privateData": {
+          "salt": "�>�����\u0014�]�NɈ",
```

Comment:
whoa @_@ does this actually work as a test vector?

Reply:
oh jeez lol. I generated this salt by doing `toString()` on a `ByteArray`. This does work, but it looks bananas. I'm down to regenerate using `toBase64Url()` instead of `toString()`.

Reply:
done